torch/csrc/jit/codegen/cuda/lower2device.cpp

```diff
@@ -356,7 +356,6 @@ kir::Expr* GpuLower::lowerExpr(const Expr* expr) {
 }
 
 GpuLower* GpuLower::current() {
-  TORCH_INTERNAL_ASSERT(active_gpu_lower != nullptr);
```

Comment:
Is this reasonable, or should we keep this assert and send an argument to the compute at map whether or not its a lowering map?